### DISCUSSION TITLE
Toggle Play/Pause for Vimeo video with enabled background mode

### DISCUSSION THREAD

**Discussion by dhwind** [vimeo/player.js#817] [Q&A]
## Toggle Play/Pause for Vimeo video with enabled background mode

Hello, I have a slider with Vimeo videos with enabled background mode. When I select some video slide, I need to pause other videos and to play video, when it's selected. Is it possible to develop such logic using this API?

**Comment by rkrishnan8594** [comment]
Yes, this behavior should be achievable with our SDK, but it might be helpful to see this example in production in order to point you in the right direction. Our player automatically pauses all videos when another starts playing. This does not work with multiple videos set to use background mode, but background mode may not be what you're looking for in this case.

**Reply by dhwind** [reply]
I tried to pause background video with your API. Had created Player instance for iframe and called `pause()` method, but didn't get an expected behaviour.
